app/src/main/java/pm/gnosis/heimdall/data/db/SolidityAddressConverter.kt

```diff
@@ -7,8 +7,8 @@ import pm.gnosis.utils.asEthereumAddressString
 
 class SolidityAddressConverter {
     @TypeConverter
-    fun fromHexString(address: String) = address.asEthereumAddress()!!
+    fun fromHexString(address: String?) = address?.asEthereumAddress()!!
```

Comment:
This doesn'T really make sense, since it will always throw if you pass `null` as an address